### ISSUE TITLE
Is batch_norm_param argument missing in the depthwise convolution 2d layer implementation?

### ISSUE THREAD

**Issue by kwotsin** [tensorflow/tensorflow#9624]
This is with reference to: https://github.com/tensorflow/tensorflow/blob/master/tensorflow/contrib/layers/python/layers/layers.py#L1850

The line says "if `batch_norm_params` is None" but there is no batch_norm_params argument included in the function, and it doesn't seem that batch_norm is implemented within the function as an option. Is the batch_norm function included in the regularizer function or has it not been implemented by default within the function?

Thanks for your help.

**Comment by drpngx** [comment]
It looks like it's a remnant of a copy-paste. Do you want to submit a PR fixing the doc?

**Comment by kwotsin** [comment]
@drpngx I have submitted a PR fixing the description error and including `batch_norm` by default as the `normalizer_fn`.

**Comment by drpngx** [comment]
Thanks! Adding cross-link: #9652.

**Comment by drpngx** [comment]
Merged. Thanks!
